packages/cli/src/zosfiles/create/blankPds/BlankPDS.definition.ts

```diff
@@ -0,0 +1,62 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandDefinition } from "@zowe/imperative";
+import { ZosFilesCreateExtraOptions, ZosFilesCreateOptions } from "../Create.options";
+import i18nTypings from "../../-strings-/en";
+
+// Does not use the import in anticipation of some internationalization work to be done later.
+const strings = (require("../../-strings-/en").default as typeof i18nTypings).CREATE;
+
+/**
+ * Create Blank PDS command definition containing its description, examples and/or options
+ * @type {ICommandDefinition}
+ */
+export const BlankPDSDefinition: ICommandDefinition = {
+    name: "data-set-blank",
```

Comment:
Have you tried the `zowe files create ds` command?
I believe it may allow for the same use case.

Reply:
Can I ask where I can find more info about `zowe files create` command?

I see docs for `zowe zos-files create` command but it takes a template argument: https://docs.zowe.org/stable/web_help/index.html?p=zowe_zos-files_create